### PR TITLE
CC: Remove check for s390x when testing cosign

### PR DESCRIPTION
**Pull request by Jordan9500**
Removed If arch is s390x in the setup cosign function

Fixes: https://github.com/kata-containers/tests/issues/5620

Depends-on: github.com/https://github.com/kata-containers/kata-containers/pull/6847

### CODE REVIEW

File: integration/confidential/fixtures/cosign/offline-fs-kbc/s390x/aa-offline_fs_kbc-resources.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
+  "default/cosign-public-key/test": "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUZrd0V3WUhLb1pJemowQ0FRWUlLb1pJemowREFRY0RRZ0FFbys5V0w1Y3lob2NlRE5HM2dlVjRoVVY4RTJMeQpNelE5SzMyalpGWWQ1UEppM1pZR1R1YXJweTJ5OFBRQkw4dE5KTjZuTldVb3d6THVjMFlReVkrVk5nPT0KLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg==",
+  "default/credential/test": ""
```
Comment (Xynnn007) [inline integration/confidential/fixtures/cosign/offline-fs-kbc/s390x/aa-offline_fs_kbc-resources.json:4]: can the test pass without `"default/credential/test"`?

File: integration/confidential/fixtures/cosign/offline-fs-kbc/x86_64/aa-offline_fs_kbc-resources.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "default/cosign-public-key/test": "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUZrd0V3WUhLb1pJemowQ0FRWUlLb1pJemowREFRY0RRZ0FFMWdIR2JmazFBcU93ZUxFTThIZlQwYm1mUUUzYgo5ZmNwL0xVNzVGTWZ4VlpYbU5WdFVwcnNITTF0aHV1aUJLT29mdjhLVjdUckZsNHA4TkpDaVhVa2hBPT0KLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg==",
+  "default/credential/test": ""
```
Comment (Xynnn007) [inline integration/confidential/fixtures/cosign/offline-fs-kbc/x86_64/aa-offline_fs_kbc-resources.json:4]: Also, can the test pass without "default/credential/test"?

**Review by Xynnn007 — APPROVED** [review]
Thanks! Overall, lgtm. only some small questions. As this test seems to be for cosign, we'd better not bring anything else into it if we can.

**Comment by stevenhorsman** [comment]
The SEV tests failed with:
```
14:12:21 Hit:1 https://download.docker.com/linux/ubuntu focal InRelease
14:12:21 Hit:3 http://us.archive.ubuntu.com/ubuntu focal InRelease
14:12:21 Ign:2 https://packages.cloud.google.com/apt kubernetes-xenial-unstable InRelease
14:12:21 Err:4 https://packages.cloud.google.com/apt kubernetes-xenial-unstable Release
14:12:21   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 142.250.189.174 443]
14:12:21 Get:5 http://us.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
14:12:22 Get:6 http://us.archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
14:12:22 Get:7 http://us.archive.ubuntu.com/ubuntu focal-security InRelease [114 kB]
14:12:23 Get:8 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [2,610 kB]
14:12:23 Get:9 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 c-n-f Metadata [16.8 kB]
14:12:23 Get:10 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1,067 kB]
14:12:23 Get:11 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 c-n-f Metadata [24.9 kB]
14:12:28 Reading package lists...
14:12:28 W: Target Packages (stable/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:50 and /etc/apt/sources.list.d/docker.list:1
14:12:28 W: Target Packages (stable/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:50 and /etc/apt/sources.list.d/docker.list:1
14:12:28 W: Target Translations (stable/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:50 and /etc/apt/sources.list.d/docker.list:1
14:12:28 W: Target Translations (stable/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:50 and /etc/apt/sources.list.d/docker.list:1
14:12:28 W: Target CNF (stable/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list:50 and /etc/apt/sources.list.d/docker.list:1
14:12:28 W: Target CNF (stable/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list:50 and /etc/apt/sources.list.d/docker.list:1
14:12:28 E: The repository 'http://apt.kubernetes.io/ kubernetes-xenial-unstable Release' no longer has a Release file.
```
so I've rebuilt them. FYI @UnmeshDeodhar

**Review by larrydewey — APPROVED** [review]
LGTM

**Review by jepio — APPROVED** [review]
LGTM

**Pull request merged**